src/rez/cli/pip.py

```diff
@@ -24,6 +24,9 @@ def setup_parser(parser, completions=False):
         "-r", "--release", action="store_true",
         help="install as released package; if not set, package is installed "
         "locally only")
+    parser.add_argument(
+         "-p", "--install_path", type=str, metavar='PATH',
```

Comment:
I think it would make sense to keep the arguments consistent across the different CLI tools. rez-build uses `-p` and `--prefix`. Is there any reason to use `--install-path` instead?

Reply:
You're right @instinct-vfx (done in 262fe07).